src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.h

```diff
@@ -15,6 +15,8 @@
 
 #include <medVtkInriaExport.h>
 
+#include <vector>
```

Comment:
is it really needed or is it a forgotten include?

Reply:
It is needed line 298.